pkg/kubelet/api/v1alpha1/runtime/api.pb.go

```diff
@@ -0,0 +1,2778 @@
+/*
```

Comment:
IMO, we should check in the generated ***go*** code because they will be consumed by other components of kubelet. It's a bit strange to expect people to write code without looking at the go interfaces.